### DISCUSSION TITLE
Deepgram api is not working for prerecorded audio file, even api playground also not working

### DISCUSSION THREAD

**Discussion by mpramakrishnareddy** [deepgram/community#774] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - STT Pre-recorded

### Details

I'm using deepgram-sdk==3.2.6 .
    
when trying to convert speech to text its giving  timeout exception.

even when tried in api playground its givng "Whoops! Something went wrong" exception

### If possible, please attach your code or paste it into the text box.

options = PrerecordedOptions(
        smart_format=True,
        model="nova-2",
        language="en-IN",
    )
response = deepgram.listen.prerecorded.v('1').transcribe_file(payload, options)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by Businesstoys** [comment]
curl -X POST \
  -H "Authorization: Token YOUR_SECRET" \
  -H 'content-type: application/json' \
  -d '{"url": "https://static.deepgram.com/examples/Bueller-Life-moves-pretty-fast.wav"}' \
  "https://api.deepgram.com/v1/listen?model=nova-2&smart_format=true"
  
The curl request is also failing .
  
<img width="570" alt="image" src="https://github.com/deepgram/community/assets/60639781/d043ff72-1b5e-402f-9511-0008e7c704e4">

**Comment by SandraRodgers** [comment, marked as answer]
Could it be possible you are affected by this issue? https://deepgram.gitbook.io/help-center/notices/cogent-tata-peering-issue-impact-on-deepgram
